app/pages/lab/workflows.spec.js

```diff
@@ -24,7 +24,7 @@ describe('WorkflowsPage', function () {
   const toggleReorderSpy = sinon.spy();
 
   before(function () {
-    wrapper = shallow(
+    wrapper = mount(
```

Comment:
Why did this change to mount?

Reply:
Because the test wasn't passing with the addition of the new `WorkflowsTable` component.

Reply:
That probably means that some of the tests should be moved to the tests for `WorkflowsTable`.

Reply:
That's a good idea, I'll do that.